[Bulk] Validation: Fixed issues in ODatadocs (#219)
MicrosoftDocs/OData-docs: Odata-docs/connectedservice/t4-templates.md

---
title: "Working with T4 templates"

> description: OData Connected Service can generate T4 templates instead of the final code.

Reviewer: ```suggestion
description: Describes how to generate T4 templates instead of the final code.
```
